app/src/main/java/xyz/malkki/neostumbler/ui/screens/MapScreen.kt

```diff
@@ -279,6 +288,36 @@ fun MapScreen(mapViewModel: MapViewModel = viewModel()) {
     }
 }
 
+private fun styleBuilder(mapStyle: State<MapViewModel.MapStyle?>, coverageParams: CoverageParams): Style.Builder {
```

Comment:
Also thinking about this, maybe it would be better to add the TileJSON layer independently from the map style?

Reply:
I do not know how to add the layer without org.maplibre.android.maps.Style. Could you give me a hint?

Reply:
I mean you need the style, but you don't have to set the base style (meaning the background map layer) and the TileJSON coverage layer at the same time. You could use [getStyle()](https://maplibre.org/maplibre-native/android/api/-map-libre%20-native%20-android/org.maplibre.android.maps/-map-libre-map/get-style.html) to get the base style and then add the coverage layer using that. Or do something similar like what is done with the hexagon layer

Reply:
Done via https://github.com/mjaakko/NeoStumbler/compare/e833de128188e3d29cbcf9d0b0a81384b0873c55..ba63cd755621c804af990f292df3d7f8e11598cb